tests/basic/pure_test.go

```diff
@@ -5130,3 +5130,125 @@ var _ = Describe("{ValidatePodNameinVolume}", func() {
 		AfterEachTest(contexts)
 	})
 })
+var _ = Describe("{MultiTenancyFATestWithPodRealm}", func() {
+	/*
+	   https://purestorage.atlassian.net/browse/PTX-24561
+	   This Test Requires Pure.json with the following requirements:
+	   -> Pure.json should have multiple FA's ,one with realm and one without realm.
+	   Following Steps are executed by the test:
+	   1.Deploys FADA applications in below combinations (for every APP provided in APP_LIST):
+	   Storage Class having pure_fa_pod_name - this pod is under a realm in one of the arrays in pure.json
+	   Storage Class having pure_fa_pod_name - this pod is not under a realm in one of the arrays in pure.json
+	   Storage Class not having pure_fa_pod_name - legacy way of FADA Apps
+	   2. Validates the application and checks if the volumes are created in the respective FA's
+	   3. Deletes the application and checks if the volumes are deleted in the respective FA's
+	*/
+	JustBeforeEach(func() {
+		StartTorpedoTest("MultiTenancyFATestWithPodRealm",
+			"MultiTenancy FATest with Pod Realm",
+			nil, 0)
+	})
+
+	itLog := "MultiTenancyFATestWithPodRealm"
+	It(itLog, func() {
+		log.InfoD(itLog)
+		var (
+			realmName           string
+			faWithRealm         *newFlashArray.Client
+			faWithoutRealm      *newFlashArray.Client
+			isRealmFAAccessible bool
+			podNameinSC         string
+			PodNameinFA         string
+		)
+		applist := Inst().AppList
+		defer func() {
+			Inst().AppList = applist
+		}()
+		Inst().AppList = []string{"fio-fa-davol"}
```

Comment:
I believe we can skip hard-coding and assert if applist is nil

Reply:
Thanks, Removed it